lib/validation/validators.js

```diff
@@ -20,6 +20,10 @@ module.exports = Validators = {
         return value !== undefined && value !== '';
     },
 
+    url: function url(value) {
+        return value === '' || Validators.regex(value, /[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b[-a-zA-Z0-9@:%_\+~#?&//=]*$/);
```

Comment:
Should this enforce an optional protocol part? at the  moment any chars are allowed at the start as there is no `^`